build_iso.sh

```diff
@@ -13,16 +13,20 @@ set -ex
 # Creating the NFSROOT
 # Removing *.profile since we don't use them
 # Removing 50-host-classes to prevent DEMO and FAIBASE to be added to the list of classes
-# Adding the Bullseye basefiles to that we deploy a Debian v11 distro
+# Adding the Bookworm basefiles to that we deploy a Debian v12 distro
+# Patches /sbin/install_packages (bug in the process of being corrected upstream)
 docker-compose -f $wd/docker-compose.yml run --rm fai-setup bash -c "\
     echo \"fai-setup -v -e -f \" && \
     fai-setup -v -e -f && \
     echo \"rm -f /ext/srv/fai/config/class/50-host-classes\" && \
     rm -f /ext/srv/fai/config/class/50-host-classes && \
     echo \"rm -f /ext/srv/fai/config/class/*.profile\" && \
     rm -f /ext/srv/fai/config/class/*.profile && \
-    echo \"wget -O /ext/srv/fai/config/basefiles/BULLSEYE64.tar.xz https://fai-project.org/download/basefiles/BULLSEYE64.tar.xz\" && \
-    wget -O /ext/srv/fai/config/basefiles/BULLSEYE64.tar.xz https://fai-project.org/download/basefiles/BULLSEYE64.tar.xz"
+    echo \"SED\" && \
+    sed -i -e \"s/-f \\\"\\\$FAI_ROOT\\/var\\/cache\\/apt\\/pkgcache\\.bin/-d \\\"\\\$FAI_ROOT\\/var\\/lib\\/apt\\/lists/\" /ext/nfsroot/sbin/install_packages && \
```

Comment:
I suggest using another delimiter than / for instance |:
`sed -i -e \"s|-f \"\$FAI_ROOT/var/cache/apt/pkgcache\.bin|-d \"\$FAI_ROOT/var/lib/apt/lists|\" /ext/nfsroot/sbin/install_packages && \`

Reply:
I think it should be: 
`sed -i -e \"s|-f \\\"\$FAI_ROOT/var/cache/apt/pkgcache\.bin|-d \\\"\$FAI_ROOT/var/lib/apt/lists|\" /ext/nfsroot/sbin/install_packages && \`

Reply:
changed in both build_iso and build_qcow2